README.md

```diff
@@ -41,7 +55,9 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- Replace this comment with your answer -->
+OK. So the variable x is assigned 4 in the first line and the variable y is assigned 3 in the second line. When these two lines run in the node REPL environment, they both return a value of undefined, but if I console.log(x) and console.log(y) they console prints out the values assigned to x and y, which are 3 and 4 respectively.
+In the third line, x is assigned the value of y. Therefore, x results to 3.
+In the fourth line, y is assigned the value of 10, to x results to 10.
```

Comment:
Understood. I wasn't clear that you wanted me to detail that there *is* no permanent relationship between x and y going forward. Suggestion for next cohort: add another sentence in your bevy of questions asking what the value of x is after y is assigned the value of 10. That may make a student think a bit deeper on this. Does that makes sense at all?